js/layout-modes/fit-rows.js

```diff
@@ -42,7 +42,7 @@ proto._getItemLayoutPosition = function( item ) {
   var itemWidth = item.size.outerWidth + this.gutter;
   // if this element cannot fit in the current row
   var containerWidth = this.isotope.size.innerWidth + this.gutter;
-  if ( this.x !== 0 && itemWidth + this.x > containerWidth ) {
+  if ( this.x !== 0 && itemWidth + this.x > containerWidth + 0.001 /*to prevent last item not fitting in some cases*/ ) {
```

Comment:
I am not sure what is a safe value for this across all browsers. 
Number.EPSILON definitely does not work. 
Using something like 0.000001 has proven to be too small as well.